packages/stylelint-config/rules/stylelint-config-standard-scss.js

```diff
@@ -0,0 +1,116 @@
+module.exports = {
+  extends: ['stylelint-config-standard-scss'].map(require.resolve),
+  rules: {
+    // Rule turned off: BEM selectors are considered violations, default syntax is `kebab-case`.
+    // Docs: https://stylelint.io/user-guide/rules/list/selector-class-pattern
+    'selector-class-pattern': null,
```

Comment:
This Stylelint configuration is based on https://github.com/lmc-eu/code-quality-tools/issues/24 opened by @adamkudrna and the proposal says that locally tested config resulted in configuration that includes `selector-class-pattern` rule with following pattern:

```
"selector-class-pattern": [
  "^lmc-([A-Z][a-zA-Z0-9]*)((--|__)[a-z0-9]+)*$",
  {
    "message": "Expected class selector to be in format `lmc-MyComponent__element`, `lmc-MyComponent--modifier`"
  }
]
```

I've tested it on https://github.com/lmc-eu/spirit-design-system and the pattern does not fit. 

There might be differences across all projects that are using https://github.com/lmc-eu/code-quality-tools and I don't think this is a breaking change we want to do right now.

Please, reviewers, give me as many feedbacks as you can on this, thank you in advance.

Reply:
Conversation solved by internal discussion.